src/key_gen/message.rs

```diff
@@ -36,21 +36,21 @@ pub enum DkgMessage<P: PublicId> {
     },
     Commitment {
         key_gen_id: u64,
-        commit: Commit,
+        commitment: Commitment,
```

Comment:
This should be exposed in the use staement above. So `use commitment::Commitment`